ethportal-api/src/types/cli.rs

```diff
@@ -181,6 +182,13 @@ pub struct TrinConfig {
     )]
     pub ws_port: u16,
 
+    #[arg(
+        long = "utp-transfer-limit", 
+        help = "The limit of max inbound and max outbound uTP transfers", 
```

Comment:
*nitpick* This is just a bit confusing to understand how the limit is actually applied. Eg is 50 the total for both inbound/outbound? Or is it 50 for inbound, 50 for outbound? What about on a per/subnetwork basis?

Maybe something like: `The limit of max uTP transfers for any given channel (inbound or outbound) for each subnetwork` ? But if you can think of a more concise way to phrase the info then 👍

Reply:
That is a great idea :+1: I will add that quick to this pr